css/bem-css.md

```diff
@@ -38,58 +40,58 @@ We do this:
 
 We _don't_ do this:
 ```css
-.feature
-.content
-.btn
+.c-feature
+.c-content
+.c-btn
 ```
 
 We do this:
 ```css
-.alert-box
-.button
+.c-alert-box
+.c-button
 ```
 
-**Element**: styles that only apply to children of a block. Elements can also be blocks themselves. Class name is a concatenation of the block name, two underscores and the element name. Examples:
-- `.alert-box__close`
-- `.button__icon`
+**Element**: styles that only apply to children of a block. Class name is a concatenation of the block name, two underscores and the element name. Examples:
+- `.c-alert-box__close`
+- `.c-button__icon`
 
 **Modifier**: override or extend the base styles of a block or element with modifier styles. Class name is a concatenation of the block (or element) name, two hyphens and the modifier name. Examples:
-- `.alert-box--success`
-- `.button--expanded`
+- `.c-alert-box--success`
+- `.c-button--expanded`
 
 ## BEM best practices
 
 Don't `@extend` block modifiers with the block base, always require that the base/unmodified class is present
```

Comment:
`Don't directly extend the base block element with a modifier, always require that the unmodified class is present`?